telemetry/telemetry.go

```diff
@@ -57,6 +58,12 @@ func main() {
 		log.Errorf("port must be > 0.")
 		return
 	}
+
+	switch {
+	case *threshold <= 0:
```

Comment:
Could you add a way to turn this threshold off? For example: if threshold == 0, then there is no limit on the number of connections.

Reply:
Good idea, I will make "-1" signify no threshold limit.

Reply:
Done